docs/mobility/slam/cartographer/_index.md

```diff
@@ -42,6 +35,16 @@ Cartographer can operate:
 
 ## Use a live machine
 
+{{% alert title="Temporarily not available in live mode" color="caution" %}}
+
+Running `cartographer` in live mode is temporarily disabled.
```

Comment:
maybe `Running cartographer with cloudslam in live mode `? The current phrasing kinda makes me think that slam just doesn't work on the robot

Reply:
Will fix! I think I was confused, thought that live mode itself was disabled?